hr_expense_invoice/views/hr_expense_views.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="hr_expense_view_view" model="ir.ui.view">
```

Comment:
"ugly" id (view_view)
please use hr_expense_view_form (optional)

Reply:
Done, thank you.